src/components/tx/SignOrExecuteForm/SignOrExecuteForm.test.tsx

```diff
@@ -539,18 +541,13 @@ describe('SignOrExecuteForm', () => {
           dispatchTxExecution: executionSpy,
         } as unknown as NullableTxSenderFunctions),
     )
+    jest.spyOn(useRemainingRelays, 'default').mockReturnValue([0, undefined, false])
```

Comment:
Please add "with the connected wallet" to the description of this test. It's easy to overlook this mock otherwise.